<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>%1 - Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="vlOptions">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabProfile">
      <attribute name="title">
       <string>Profiles</string>
      </attribute>
      <layout class="QVBoxLayout" name="vlProfile">
       <item>
        <widget class="QGroupBox" name="gbWidgets">
         <property name="title">
          <string>Content Open/Select Mode</string>
         </property>
         <layout class="QVBoxLayout" name="vlProfileContentMode">
          <item>
           <widget class="QRadioButton" name="rbModern">
            <property name="text">
             <string notr="true">%1 1.9+</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbClassic">
            <property name="text">
             <string notr="true">%1 1.0-1.8</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbDoubleclick">
            <property name="text">
             <string>Open with Doubleclick</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbDefaultProfile">
         <property name="title">
          <string>Default Profile</string>
         </property>
         <layout class="QVBoxLayout" name="vlDefaultProfile">
          <item>
           <widget class="QComboBox" name="cbProfiles"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbDefaultFolder">
         <property name="title">
          <string>Custom GTA V Folder</string>
         </property>
         <layout class="QVBoxLayout" name="vlCustomGTAVFolder">
          <item>
           <widget class="QCheckBox" name="cbForceCustomFolder">
            <property name="text">
             <string>Force using Custom Folder</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hlDefaultFolder">
            <item>
             <widget class="QLineEdit" name="txtFolder"/>
            </item>
            <item>
             <widget class="QToolButton" name="cmdExploreFolder">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="vsProfile">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPictures">
      <attribute name="title">
       <string>Pictures</string>
      </attribute>
      <layout class="QVBoxLayout" name="vlTabPictures">
       <item>
        <widget class="QGroupBox" name="gbPicResolution">
         <property name="title">
          <string>Export Size</string>
         </property>
         <layout class="QVBoxLayout" name="vlGbPicRes">
          <item>
           <widget class="QRadioButton" name="rbPicDefaultRes">
            <property name="text">
             <string>Default: %1x%2</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbPicDesktopRes">
            <property name="text">
             <string>Screen Resolution: %1x%2</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hlCustomRes">
            <item>
             <widget class="QRadioButton" name="rbPicCustomRes">
              <property name="text">
               <string>Custom Size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labPicCustomRes">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Custom Size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbPicExportWidth">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>3840</number>
              </property>
              <property name="value">
               <number>960</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labPicXDescription">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbPicExportHeight">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>2160</number>
              </property>
              <property name="value">
               <number>536</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="hsPicCustomSize">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hlAspectRatio">
            <item>
             <widget class="QCheckBox" name="cbIgnoreAspectRatio">
              <property name="text">
               <string>Ignore Aspect Ratio</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbPicQuality">
         <property name="title">
          <string>Export Quality</string>
         </property>
         <layout class="QVBoxLayout" name="vlHlPicQuality">
          <item>
           <widget class="QCheckBox" name="cbPicCustomQuality">
            <property name="text">
             <string>Enable Custom Quality</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hlPicQuality">
            <item>
             <widget class="QLabel" name="labPicQualityDescription">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Quality:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="hsPicQuality">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labPicQuality">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>%1%</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbSnapmaticPictureViewer">
         <property name="title">
          <string>Picture Viewer</string>
         </property>
         <layout class="QVBoxLayout" name="vlSnapmaticPictureViewer">
          <item>
           <widget class="QCheckBox" name="cbSnapmaticNavigationBar">
            <property name="text">
             <string>Enable Navigation Bar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="vsPictures">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPlayers">
      <attribute name="title">
       <string>Players</string>
      </attribute>
      <layout class="QVBoxLayout" name="vlPlayers">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="twPlayers">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGame">
      <attribute name="title">
       <string>Game</string>
      </attribute>
      <layout class="QVBoxLayout" name="vlGame">
       <item>
        <widget class="QGroupBox" name="gbSocialClub">
         <property name="title">
          <string>Social Club Version</string>
         </property>
         <layout class="QVBoxLayout" name="vlGameSocialClub">
          <item>
           <widget class="QLabel" name="labSocialClubFound">
            <property name="text">
             <string>Found: %1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labSocialClubLanguage">
            <property name="text">
             <string>Language: %1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbSteam">
         <property name="title">
          <string>Steam Version</string>
         </property>
         <layout class="QVBoxLayout" name="vlGameSteam">
          <item>
           <widget class="QLabel" name="labSteamFound">
            <property name="text">
             <string>Found: %1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labSteamLanguage">
            <property name="text">
             <string>Language: %1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="vsGame">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabStats">
      <attribute name="title">
       <string>Feedback</string>
      </attribute>
      <layout class="QVBoxLayout" name="vlStats">
       <item>
        <widget class="QGroupBox" name="gbUserStats">
         <property name="title">
          <string>Participation</string>
         </property>
         <layout class="QVBoxLayout" name="vlUserStats">
          <item>
           <widget class="QCheckBox" name="cbParticipateStats">
            <property name="text">
             <string>Participate in %1 User Statistics</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labUserStats">
            <property name="text">
             <string notr="true">&lt;a href=&quot;%2&quot;&gt;%1&lt;/a&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbCategories">
         <property name="title">
          <string>Categories</string>
         </property>
         <layout class="QVBoxLayout" name="vlCategories">
          <item>
           <widget class="QCheckBox" name="cbGeneralStats">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Hardware, Application and OS Specification</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbOSLangStats">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>System Language Configuration</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbAppConfigStats">
            <property name="text">
             <string>Application Configuration</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbUsageData">
            <property name="text">
             <string>Personal Usage Data</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbOther">
         <property name="title">
          <string>Other</string>
         </property>
         <layout class="QVBoxLayout" name="vlFeedbackOther">
          <item>
           <layout class="QHBoxLayout" name="hlParticipation">
            <item>
             <widget class="QLabel" name="labParticipationID">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Participation ID: %1</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdCopyStatsID">
              <property name="text">
               <string>&amp;Copy</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="vsUserStats">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInterface">
      <attribute name="title">
       <string>Interface</string>
      </attribute>
      <layout class="QVBoxLayout" name="vlInterface">
       <item>
        <widget class="QGroupBox" name="gbLanguage">
         <property name="title">
          <string>Language for Interface</string>
         </property>
         <layout class="QVBoxLayout" name="vlLanguage">
          <item>
           <widget class="QComboBox" name="cbLanguage"/>
          </item>
          <item>
           <widget class="QLabel" name="labCurrentLanguage">
            <property name="text">
             <string>Current: %1</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbAreas">
         <property name="title">
          <string>Language for Areas</string>
         </property>
         <layout class="QVBoxLayout" name="vlAreas">
          <item>
           <widget class="QComboBox" name="cbAreaLanguage"/>
          </item>
          <item>
           <widget class="QLabel" name="labCurrentAreaLanguage">
            <property name="text">
             <string>Current: %1</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbStyle">
         <property name="title">
          <string>Style</string>
         </property>
         <layout class="QVBoxLayout" name="vlStyle">
          <item>
           <widget class="QCheckBox" name="cbDefaultStyle">
            <property name="text">
             <string>Use Default Style (Restart)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hlStyle">
            <item>
             <widget class="QLabel" name="labStyle">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Style:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbStyleList">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbFont">
         <property name="title">
          <string>Font</string>
         </property>
         <layout class="QVBoxLayout" name="vlFont">
          <item>
           <widget class="QCheckBox" name="cbDefaultFont">
            <property name="text">
             <string>Use Default Font (Restart)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hlFont">
            <item>
             <widget class="QLabel" name="labFont">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Font:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFontComboBox" name="cbFont">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="cmdFont">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string notr="true">...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="vsInterface">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlButtons">
     <item>
      <spacer name="hsButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdOK">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Apply changes</string>
       </property>
       <property name="text">
        <string extracomment="OK, Cancel, Apply">&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Discard changes</string>
       </property>
       <property name="text">
        <string extracomment="OK, Cancel, Apply">&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdCancel</sender>
   <signal>clicked()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
